### ISSUE TITLE
Symmetry-guided gradient descent for quantum neural networks

### ISSUE THREAD

**Issue by github-nakasho** [github-nakasho/quantph#8164]
# 論文概要

多くの教師あり学習は、画像分類における並進対称性などを持つことから、対称性を持つ制約を簡潔な数学的定式化することを提案。ここからコスト関数に制約を適用する2つの方法を設計した。回路ansatzを変更する従来手法と比較して実装が簡単。

# 論文を理解する上で重要な図など

<img width="505" alt="Screenshot 2024-04-10 at 3 11 28 PM" src="https://github.com/github-nakasho/quantph/assets/32871721/ce36d1f8-be24-4e1c-9108-ff52243fb48f">
<img width="514" alt="Screenshot 2024-04-10 at 3 11 38 PM" src="https://github.com/github-nakasho/quantph/assets/32871721/03251fbe-ecfd-438c-b2b5-416c1fbdefe4">
<img width="514" alt="Screenshot 2024-04-10 at 3 11 48 PM" src="https://github.com/github-nakasho/quantph/assets/32871721/4648d610-7ed0-4f8d-acd1-f865918e549c">

# 論文リンク

[https://arxiv.org/abs/2404.06108](https://t.co/ne1HEnLqyi)
